Backend/app/rider-platform/rider-app/Main/src/Storage/Tabular/Rating.hs

```diff
@@ -0,0 +1,67 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Storage.Tabular.Rating where
+
+import qualified Domain.Types.Rating as Domain
+import Kernel.Prelude
+import Kernel.Storage.Esqueleto
+import Kernel.Types.Id
+import Storage.Tabular.Person (PersonTId)
+import Storage.Tabular.Ride (RideTId)
+
+mkPersist
+  defaultSqlSettings
+  [defaultQQ|
+    RatingT sql=rating
+      id Text
+      rideId RideTId
+      riderId PersonTId
+      ratingValue Int
```

Comment:
maybe type

Reply:
> maybe type

for what? Why someone might want to rate ride as null?